packages/@snowpack/plugin-webpack/plugin.js

```diff
@@ -33,9 +33,13 @@ module.exports = function plugin(config, args) {
     throw new Error("Output Pattern for CSS must end in .css");
   }
 
+  // Webpack handles minification, so it's safe to disable Snowpack's default minifier.
+  config.buildOptions.minify = false;
+
   return {
     defaultBuildScript: "bundle:*",
-    async bundle({ srcDirectory, destDirectory, log, jsFilePaths }) {
+    async bundle({ srcDirectory, destDirectory, log }) {
```

Comment:
keeping the older interface right now to prove backwards compatibility, can update all plugins in a single pass before releasing 2.7.0